resources/analysisTools/snvPipeline/filter_PEoverlap.py

```diff
@@ -96,18 +96,23 @@ def decreaseDP4(remove_base, remove_is_reverse, REF, ALT, DP4rf, DP4rr, DP4af, D
 class BoolCounter:
     """ A class for counter objects """ 
 
-    def __init__(self):
-        self.ref = 0
-        self.alt = 0
+    def __init__(self, ref, alt):

```

Comment:
What are ref and alt? Are these strings/chars representing bases? If so you could name them `refBase`, which would make this **much** clearer. Capitalized variables are for constants, or not?